.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,49 @@
-<!--- 
-Thanks for contributing to manim!
-**Please ensure that your pull request works with the latest version of manim from this repository.**
-You should also include:
-  1. The motivation for making this change (or link the relevant issues)
-  2. How you tested the new behavior (e.g. a minimal working example, before/after
-     screenshots, gifs, commands, etc.) This is rather informal at the moment, but
-     the goal is to show us how you know the pull request works as intended.
-If you don't need any of the optional sections, feel free to delete them to prevent clutter.
--->
-
-## List of Changes
-<!-- List out your changes one by one like this:
-- Change 1
-- Change 2
-- and so on..
+<!--
+Thank you for contributing to manim!
 
-Be sure to note your changes in the [changelog](docs/source/changelog.rst) if your
-changes warrant it!
+Please ensure that your pull request works with the latest
+version of manim from this repository.
 -->
 
 ## Motivation
-<!-- Why you feel your changes are required. -->
+<!-- Outline your motiviation: why do you feel that your
+changes are required? -->
+
+## Overview / Explanation for Changes
+<!-- Give an overview of your changes and explain how they 
+resolve the situation described in the previous section.
+
+For PRs introducing new features, please provide code snippets
+using the newly introduced functionality and ideally even the
+expected rendered output. -->
 
-## Explanation for Changes
-<!-- How do your changes solve aforementioned problems? -->
+## Oneline Summary of Changes
+<!-- Please update the lines below with a oneline summary 
+for your changes. It will be included in the list of upcoming changes at 
+https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release -->
+```
+- Added new feature ... / Fixed bug ... / ... (:pr:`PR NUMBER HERE`)
+```
 
 ## Testing Status
-<!-- Optional, but recommended, your computer specs and what tests you ran with their results, if any -->
+<!-- Optional (but recommended): your computer specs and
+what tests you ran with their results, if any. This section
+is also intended for other testing-related comments. -->
 
 ## Further Comments
-<!-- Optional, any edits/updates should preferably be written here. -->
+<!-- Optional, any further comments regarding your PR
+that might be useful for reviewers.. -->
 
 ## Acknowledgements
 - [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
-- [ ] I have added an entry describing the changes from this PR to the [Changelog](https://docs.manim.community/en/latest/changelog.html) at `docs/source/changelog.rst`
 
 <!-- Once again, thanks for helping out by contributing to manim! -->
+
+
+<!-- Do not modify the lines below. -->
+## Reviewer Checklist
+- [ ] Newly added functions/classes are either private or have a docstring
+- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
```

Comment:
<!-- If your review is requested but you don't have time for it, please leave a small note on that -->